t/t7519-status-fsmonitor.sh

```diff
@@ -354,4 +354,16 @@ test_expect_success 'discard_index() also discards fsmonitor info' '
 	test_cmp expect actual
 '
 
+# Use test files that start with 'z' so that the entries being added
+# and removed appear at the end of the index.
+test_expect_success 'status succeeds and after staging/unstaging ' '
```

Comment:
nit: do you mean to have "and" here? Is it extra, or are you missing a word after?

Reply:
(Looking at the "Alternate testing approach" commit, it looks like it should be dropped along with the word "before")

Reply:
Nice catch thanks!

Reply:
@derrickstolee I will push a fix for this and squash all commits before merging.  I'm holding the PR right now to hear @dscho's on the other thread regarding this test.

Reply:
Oh, from my side it looks good, too!